src/Param.cc

```diff
@@ -543,8 +641,15 @@ bool Param::ValueFromString(const std::string &_value)
     {
       if (!tmp.empty())
       {
-        return ParseUsingStringStream<ignition::math::Pose3d>(
-            tmp, this->dataPtr->key, this->dataPtr->value);
+        const ElementPtr p = this->dataPtr->parentElement.lock();
+        if (!_ignoreAttributes && p)
+        {
+          const Param_V attributes = p->GetAttributes();
```

Comment:
Consider making `attributes` a const ref to avoid copying the vector of attributes (assuming `GetAttributes` also returns a const ref per my previous comment)

Reply:
https://github.com/ignitionrobotics/sdformat/pull/589/commits/807b821a12a3824e78596056f6e78ffb96f01126